Add information about row mapping and cell retention measurements (#106)
antmicro/litex-rowhammer-tester: doc/general.rst

   Bit-flips for row   132: 12
   Bit-flips for row   134: 3

Row mapping:

> Map row pairs from row 3 (``--start-row``) to row 60 (``--nrows``) where the next pair is 5 rows away (``--row-jump``) from the previous one:

Reviewer: Maybe we should specify that `nrows` sets the total number of rows, but the rows are counted from 0 to `nrows`-1:
```suggestion
Map row pairs from row 3 (``--start-row``) to row 59 (``--nrows`` - 1, as we count total number of rows from 0) where the next pair is 5 rows away (``--row-jump``) from the previous one:
```

.. code-block:: sh

  (venv) $ python hw_rowhammer.py --nrows 60 --pattern 01_in_row --all-rows --start-row 3 --row-jump 5 --no-refresh --read_count 10e4

> Map row pairs from row 3 to to row 60 without a distance between subsequent pairs (no ``--row-jump``), which means that rows pairs are incremented by 1:

Reviewer: ```suggestion
Map row pairs from row 3 to to row 59 without a distance between subsequent pairs (no ``--row-jump``), which means that rows pairs are incremented by 1:
```

Author: Done

.. code-block:: sh

  (venv) $ python hw_rowhammer.py --nrows 60 --pattern 01_in_row --all-rows --start-row 3 --no-refresh --read_count 10e4

> Map all row pairs (from 0 to nrows):

Reviewer: ```suggestion
Map all row pairs (from 0 to nrows - 1):
```

Author: Done

.. code-block:: sh

  (venv) $ python hw_rowhammer.py --nrows 512 --pattern 01_in_row --all-rows --no-refresh --read_count 10e4

Cell retention measurement:

Perform set of tests for different read count values in a given range for one row pair (50, 100):

.. code-block:: sh

  (venv) $ python hw_rowhammer.py --nrows 512 --pattern 01_in_row --row-pairs const --const-rows-pair 50 100 --no-refresh --read_count 10e4 10e5 20e4

Perform set of tests for different read count values in a given range for a sequence of attacks for different pairs, where the first row of a pair is 40 and the second one is a row of a number from range (40, nrows - 1):

.. code-block:: sh

  (venv) $ python hw_rowhammer.py --nrows 512 --pattern 01_in_row --row-pairs sequential --start-row 40 --no-refresh --read_count 10e4 10e5 20e4

> Map all row pairs (from 0 to nrows) and perform a set of tests for different read count values, starting from 10e4 and ending at 10e5 with a step of 20e4 (``--read_count_range [start stop step]``):

Reviewer: ```suggestion
Map all row pairs (from 0 to nrows - 1) and perform a set of tests for different read count values, starting from 10e4 and ending at 10e5 with a step of 20e4 (``--read_count_range [start stop step]``):
```

Author: Done
